src/Auth0.OidcClient.Shared/Auth0ClientOptions.cs

```diff
@@ -49,6 +50,30 @@ public class Auth0ClientOptions
         /// </summary>
         public string Scope { get; set; }
 
+        /// <summary>
+        /// Allow overriding the RetryMessageHandler
+        /// <example>
+        /// var handler = new HttpClientHandler();
+        /// var options = new Auth0ClientOptions
+        /// {
+        ///    RefreshTokenMessageHandler = handler
+        /// };
+        /// </example>
+        /// </summary>
```

Comment:
Summary tag should close before the example tag.

Reply:
I've fixed this with another push sorry about that